extensions/wikia/MyHome/MyHome.class.php

```diff
@@ -140,13 +140,13 @@ public static function getSectionName($editor, $text, $section, &$error) {
 		global $wgParser;
 
 		// make sure to properly init this variable
-		self::$editedSectionName = false;
+		static::$editedSectionName = false;
 
 		// check for section edit
 		if (is_numeric($section)) {
 			$hasmatch = preg_match( "/^ *([=]{1,6})(.*?)(\\1) *\\n/i", $editor->textbox1, $matches );
 
-			if ( $hasmatch and strlen($matches[2]) > 0 ) {
+			if ( $hasmatch && strlen($matches[2]) > 0 ) {
 				// this will be saved in recentchanges table in MyHome::storeInRecentChanges
 				self::$editedSectionName = $wgParser->stripSectionName($matches[2]);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)